src/components/TablePerson/TablePerson.tsx

```diff
@@ -0,0 +1,44 @@
+import classNames from 'classnames';
+import { Person } from '../../types';
+import { useParams } from 'react-router-dom';
+import { PersonLink } from '../PersonLink';
+
+type Props = {
+  person: Person;
+};
+export const TablePerson: React.FC<Props> = ({ person }) => {
+  const { slug } = useParams();
+  const { name, sex, born, died, motherName, fatherName, mother, father } =
+    person;
+
+  return (
+    <tr
+      data-cy="person"
+      className={classNames('', {
+        'has-background-warning': person.slug === slug,
+      })}
```

Comment:
do u need this empty line?
```suggestion
      className={classNames({
        'has-background-warning': person.slug === slug,
      })}
```

Reply:
done